specification/trace/semantic_conventions/cloudevents.md

```diff
@@ -13,8 +18,33 @@ The attributes described in this document are not specific to a particular opera
 
 <!-- tocstop -->
 
+## Spans
+
+### Creation
+
+Instrumentation SHOULD create a new span and populate the [Distributed Tracing Extension](https://github.com/cloudevents/spec/blob/v1.0.1/extensions/distributed-tracing.md) on the event. This applies when:
+
+- CloudEvent is created by instrumented library. It may be impossible or impractical to create span in CloudEvent constructor (or factory) - instrumentation MAY create span later when passing event to transport layer.
```

Comment:
```suggestion
- A CloudEvent is created by the instrumented library. It may be impossible or impractical to create the Span during event creation (e.g. inside the constructor or in a factory method) - instrumentation MAY create the Span later, when passing the event to the transport layer.
```